nilearn/masking.py

```diff
@@ -513,6 +513,139 @@ def compute_multi_background_mask(data_imgs, border_size=2, upper_cutoff=0.85,
     return mask
 
 
+def compute_gray_matter_mask(target_img, threshold=.5,
+                             connected=True, opening=2, memory=None,
+                             verbose=0):
+    """ Compute a mask corresponding to the gray matter part of the brain.
+    The gray matter part is calculated through the resampling of MNI152
+    template Gray Matter mask onto the image
+
+    Parameters
+    ----------
+    target_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Images used to compute the mask. 3D and 4D images are accepted.
+        Only the shape and affine of target_img will be used here.
+
+    threshold: float, optional
+        The value under which the MNI template is cut off.
+
+    connected: boolean, optional
+        if connected is True, only the largest connect component is kept.
+
+    opening: bool or int, optional
+        if opening is True, a morphological opening is performed, to keep
+        only large structures. This step is useful to remove parts of
+        the skull that might have been included.
```

Comment:
Sorry, but you may want to remove this sentence as there is no problem with skulls being segmented here, as we used a well-defined template.